portal-api/src/v1/users/sign-in-link.service.test.js

```diff
@@ -112,6 +132,10 @@ describe('SignInLinkService', () => {
               .mockResolvedValueOnce(undefined);
           });
 
+          afterEach(() => {
+            SIGN_IN_LINK_DURATION.MINUTES = originalSignInLinkDurationMinutes;
```

Comment:
If we had supplied the duration as a variable through the constructor instead of a direct import (i.e. a global value), it would have been easier to write tests (we wouldn't have needed this oddity around editing a "constant")

I'm happy for you to leave it for this PR, but watch out for it in future changes. I know nearly all of DTFS is doing it this way currently but I think it's an area we should gradually improve as we add / change code